doc/dependencies.md

```diff
@@ -28,3 +28,18 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | xkbcommon |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk#L86) (Linux only) |
 | ZeroMQ | [4.3.1](https://github.com/zeromq/libzmq/releases) | 4.0.0 | No |  |  |
 | zlib | [1.2.11](https://zlib.net/) |  |  |  | No |
+
+Controlling dependencies
+------------------------
+Some dependencies are not needed in all configurations. The following are some factors that affect the dependency list.
+
+#### Options passed to `./configure`
+* **miniupnpc** is not needed with  `--with-miniupnpc=no`.
+* **berkeley-db4** is not needed with `--disable-wallet`.
+* **protobuf** is not needed with `--disable-bip70`.
+* **qt** is not needed with `--without-gui`.
+* If the **qrencode** dependency is absent, QR support won't be added. To force an error when that happens, pass `--with-qrencode`.
+* **zeromq** is needed only with the `--with-zmq` option.
+
+#### Other
+* **librsvg** is only needed if you need to run `make deploy` on (cross-compliation to) macOS.
```

Comment:
`librsvg` is not listed in the table. Should it be added?

Reply:
Good catch. It was added in an earlier commit, but hence forgotten... Will update.